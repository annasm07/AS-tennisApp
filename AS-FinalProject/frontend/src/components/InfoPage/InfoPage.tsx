import * as React from 'react';
import {Text} from 'react-native';
import {SafeAreaView} from 'react-native-safe-area-context';

export default function InfoPage() {
  return (
    <SafeAreaView>
      <Text>Info Page LIST PAGE IS WORKING</Text>
    </SafeAreaView>
  );
}
