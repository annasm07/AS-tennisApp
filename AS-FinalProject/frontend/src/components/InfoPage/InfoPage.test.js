import {render} from '@testing-library/react-native';
import React from 'react';
import ConfigureStore from 'redux-mock-store';
import {Provider} from 'react-redux';
import InfoPage from './InfoPage';

describe('When invoked a InfoPage component', () => {
  const mockStore = ConfigureStore();
  const store = mockStore({
    user: {user: {playerId: '123cde'}},
  });
  test('Should render InfoPage component', () => {
    const infoPage = render(
      <Provider store={store}>
        <InfoPage />
      </Provider>,
    );
    expect(infoPage).toMatchSnapshot();
  });
});
