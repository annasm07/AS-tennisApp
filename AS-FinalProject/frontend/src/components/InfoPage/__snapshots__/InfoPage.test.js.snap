// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When invoked a InfoPage component Should render InfoPage component 1`] = `
<RNCSafeAreaView>
  <Text>
    Info Page LIST PAGE IS WORKING
  </Text>
</RNCSafeAreaView>
`;
